polygon/heimdall/service_test.go

```diff
@@ -152,8 +152,8 @@ func (suite *ServiceTestSuite) SetupSuite() {
 	tempDir := suite.T().TempDir()
 	dataDir := fmt.Sprintf("%s/datadir", tempDir)
 	logger := testlog.Logger(suite.T(), log.LvlCrit)
-	store := NewMdbxServiceStore(logger, dataDir, tempDir, 1)
-	borConfig := suite.chainConfig.Bor.(*borcfg.BorConfig)
+	store := NewMdbxStore(logger, dataDir, 1)
+	borConfig := params.AmoyChainConfig.Bor.(*borcfg.BorConfig)
```

Comment:
The chain config here needs to come from the test suite since there is a test that relies on bor mainnet data and another test that relies on Amoy data. Please change it back to how it was.